javascript/modules/client/src/client-addressList/internal/client/decoding.ts

```diff
@@ -1,17 +1,18 @@
 import { bytesToHex } from "@aragon/sdk-common";
```

Comment:
Do we need to have a path like `<plugin>/internal/client/decoding.ts`? In my opinion `/client` is already implied.... unless there were too many files on the `internal` folder of that plugin, I would flatten it

Reply:
Yes thats a different thing, `/internal/client/encoding.ts` refers to the encoding class of the client. Maybe it's a bit weird due to the naming, but they are totally different thing and should be separated imo. Maybe we could reconsider the `modules` naming instead of client

Reply:
ok, how about naming the plugin like `plugin-erc20` and `plugin-addressList`?